src/pages/using-spark/foundations/color-usage.mdx

```diff
@@ -270,7 +270,7 @@ import {
     <input
       class="sprk-b-TextInput sprk-u-Width-100"
       id="text-input"
-      value="Focus to see styling"
+      defaultValue="Focus to see styling"
```

Comment:
This was causing a bug in the browser console so i fixed it. You can't pass a value in React without also passing an onChange handler. In cases where you want it to be mutable you need to use defaultValue.